{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737703769686, "stop": 1737703769686}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "driver", "value": "'Edge'"}], "start": 1737703769211, "stop": 1737703769760, "uuid": "b4cc0aca-6731-4a2b-9ce5-bc0a23853fd8", "historyId": "e664392ae8e96150bc26ed53191a48bb", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az2032-218"}, {"name": "thread", "value": "1943-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}